### ISSUE TITLE
Idea: Backup signups

### ISSUE THREAD

**Issue by Kakirtog** [wwevo/guildhome#61]
Make a second queue for signups, so that people can still sign up when the max number of signups is reached. Should people sign out again, like with the last raid, the organiser can either choose from the backups or the first one in the backup queue gets moved to the main event. Should give some form of notice when that happens maybe.

**Comment by AniOnGit** [comment]
Well, we already have a function to leave signups open beyond maximal. I guess what's now missing is the possibility to see which of the signups was after the maximum was reached.
To keep it simple I'd suggest putting the signups general in chronological order (if not already) and then add a second line 'Queued:' below where the members are put which signed up after cap?

**Comment by Kakirtog** [comment]
Yeah, that's mainly the idea I had, my wording was a little off I guess.
